docs/admin/react-hooks.mdx

```diff
@@ -711,23 +711,41 @@ const CustomComponent: React.FC = () => {
 
-| Property                  | Description                                                                                                        |
-| ------------------------- | ------------------------------------------------------------------------------------------------------------------ |
-| **`currentEditor`**       | The user currently editing the document.                                                                             |
-| **`docConfig`**              | Either the Collection or Global config of the document, depending on what is being edited.                         |
-| **`documentIsLocked`**    | Whether the document is currently locked by another user.                                                          |
-| **`id`**                  | If the doc is a collection, its ID will be returned                                                                |
-| **`getDocPermissions`**   | Method to retrieve document-level user preferences.                                                 |
-| **`getDocPreferences`**   | Method to retrieve document-level user preferences.                                                 |
-| **`hasPublishedDoc`**     | Whether the document has a published version.                                                                      |
-| **`incrementVersionCount`** | Method to increment the version count of the document.                                                           |
-| **`preferencesKey`**      | The `preferences` key to use when interacting with document-level user preferences.                                 |
-| **`versions`**            | Versions of the current doc.                                                                                        |
-| **`unpublishedVersions`** | Unpublished versions of the current doc.                                                                            |
-| **`publishedDoc`**        | The currently published version of the doc being edited.                                                            |
-| **`getVersions`**         | Method to retrieve document versions.                                                               |
-| **`docPermissions`**      | The current documents permissions. Collection document permissions fallback when no id is present (i.e. on create). |
-| **`versionCount`**        | The current version count of the document.                                                                          |
+| Property                            | Description                                                                                             |
+| ----------------------------------- | ------------------------------------------------------------------------------------------------------- |
+| **`action`**                        | The current action being performed on the document (create, edit, etc.).                                |
```

Comment:
This description isn't quite right,`action` here describes the URL attached to the `action` attribute on the underlying `form` element. This specifies where to send the form data when the form is submitted.